File: problemas/iniciante/2166/2166.lua
n = tonumber(io.read(), 10)

resposta = 0.0
if (n == 0) then
  resposta = 1.0
else
  resposta = 2.0
  for _ = 2, n do
    resposta = 2.0 + 1.0 / resposta
  end
  resposta = 1.0 + 1.0 / resposta
end

print(string.format("%.10f", resposta))

File: problemas/iniciante/2166/2166.swift
import Foundation

let n = Int(readLine(strippingNewline: true)!)!

var resposta = 0.0
if(n == 0){
  resposta = 1.0
}else{
  resposta = 2.0
  for _ in 1..<n {
    resposta = 2.0 + 1.0 / resposta
  }
  resposta = 1.0 + 1.0 / resposta
}

print(String(format: "%.10f", resposta))